packages/nextjs/hooks/scaffold-eth/useEthPrice.ts

```diff
@@ -28,5 +27,24 @@ export const useEthPrice = () => {
     fetchPriceFromUniswap();
   }, [provider]);
 
+  useInterval(
+    () => {
+      const fetchPriceFromUniswap = async () => {
+        try {
+          const DAI = new Token(1, "0x6B175474E89094C44Da98b954EedeAC495271d0F", 18);
+          const pair = await Fetcher.fetchPairData(DAI, WETH[1], provider);
```

Comment:
instead of `WETH[1]` we can keep ` WETH[DAI.chainId]` as it was previously since it describes itself instead of putting a magic number `1` 🙌